internal/pkg/db/db.go

```diff
@@ -238,6 +270,30 @@ func (backend *Backend) FilterDocumentsByTag(documents []*sbom.Document, tags ..
 	return documents, nil
 }
 
+func (backend *Backend) MoveAlias(revision, base *sbom.Document) error {
+	baseUUID := base.GetMetadata().GetId()
```

Comment:
This isn't a UUID, it's the metadata ID string. Currently it's unique since it's the primary key but I think the eventual goal is to change that. Might need another update to the storage backend to allow for operations on UUIDs

If a UUID is what you need you should be able to call

```go
baseUUID, err := ent.GenerateUUID(base)
```